ui/app/styles/components/replication-dashboard.scss

```diff
@@ -70,9 +65,9 @@
         grid-row: 3/3;
       }
     }
-  }
 
-  .has-text-highlight {
-    color: $yellow-500;
+    .has-border-danger {
```

Comment:
could you move this into the `helpers.scss` too? we'll want to use it on the other dashboards :D